roles/htcondor/tasks/install_other_required_packages.yaml

```diff
@@ -0,0 +1,35 @@
+---
+# install_other_required_packages.yaml
+
+- name: install packages required for python3
+  yum:
+    name:
+      - python3
+      - python3-devel
+      - python3-pip
+    state: installed
+
+- name: enable CRB for EL9
+  command: dnf config-manager --set-enabled crb
+  args:
+    warn: false
+  when: ansible_facts['distribution_major_version'] == "9"
+
+# TODO see if xorg-x11-apps is really needed
+- name: install other required packages
+  package:
+    name: ['@development', 's-nail', 'xauth', 'xorg-x11-fonts-Type1']
```

Comment:
What is the @development package?
Make xauth/x11 pieces option via vars file.

Reply:
@development seems to be a group of development tools like git, libtool, etc.

Reply:
Why would @development and any of the other packages be needed by condor?

Reply:
Removing the package seemed to cause a few errors with the poller but installing just the gcc and git packages in the @development package seemed to resolve them. I can switch the install to just those two packages if that would be better